BoxBrowseSDK/BoxBrowseSDK/Categories/NSBundle+BOXBrowseSDKAdditions.m

```diff
@@ -13,14 +13,9 @@ @implementation NSBundle (BOXBrowseSDKAdditions)
 
 + (NSBundle *)boxBrowseSDKResourcesBundle
 {
-    static NSBundle *browseSDKResourceBundle = nil;
-    static dispatch_once_t predicate;
-    dispatch_once(&predicate, ^{
-        NSBundle *browseSDKBundle = [NSBundle bundleForClass:[BOXBrowseSDKResourceLocator class]];
-        NSString *browseSDKResourceBundlePath = [browseSDKBundle.bundlePath stringByAppendingPathComponent:@"BoxBrowseSDKResources.bundle"];
-        browseSDKResourceBundle = [NSBundle bundleWithPath:browseSDKResourceBundlePath];
-    });
-
+    NSBundle *browseSDKBundle = [NSBundle bundleForClass:[BOXBrowseSDKResourceLocator class]];
```

Comment:
Not sure I understand how that could work since this is a synchronous method. Can you give me the pseudo code?

Agree that we need to be careful about performance with this. I'll do some timing checks to ensure we're not adding too much.